athf/mcp/tools/agent_tools.py

```
@@ -0,0 +1,151 @@
+
+        result = agent.execute(input_data)
+        if not result.success:
+            return _json_result({"error": result.error or "Research failed"})
+
+        output = result.data
+        if output is None:
+            return _json_result({"error": "No output from researcher"})
+
+        # Build a report from the skill outputs
+        report_parts = [
+            "# {topic} Research\n".format(topic=topic),
```

Ruff (0.15.6)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)